xla/service/gpu/ir_emitter_unnested.cc

```diff
@@ -2647,6 +2648,37 @@ absl::Status IrEmitterUnnested::EmitHloInstruction(
               streams.destination_stream_id, streams.source_stream_id));
           return EmitFusion(Cast<HloFusionInstruction>(wrapped));
         }
+        case HloOpcode::kCall: {
+          const ExecutionStreamAssignment& stream_assignment =
```

Comment:
This is pretty long, let's use a helper function (`return Emit...`) instead to decrease this function's cognitive complexity!

Reply:
I broke out the code into `EmitAsyncComputation`.